Symfony/CS/Tests/Fixer/PSR2/EofEndingFixerTest.php

```diff
@@ -52,17 +52,46 @@ public function provideCases()
 ',
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;",
+                "<?php\r\$a = 4;\n",
+                "<?php\r\$a = 4;",
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;\r",
+                "<?php\r\$a = 5;\n",
+                "<?php\r\$a = 5;\r",
```

Comment:
Following PSR 2 `\r` should be replaced by `\n`, unless within `T_INLINE_HTML`.
I'm not sure what test case you would like me to add here.